README.md

```diff
@@ -3,6 +3,13 @@
 Spectro Cloud module is a container for all Palette resources that are used together. 
 You can use modules to create lightweight abstractions, using yaml files translated into Terraform.
 
+## Disclaimer
+
+This repository serves as a demonstration example for utilizing the SpectroCloud Terraform provider to create resources using YAML as a source. 
+Please note that SpectroCloud does not actively maintain or provide modules support for all active Spectrocloud terraform provider releases.
+
+Feel free to fork this repository and customize it to meet your specific requirements.
+
```

Comment:
This repository serves as a demonstration for utilizing ...
SpectroCloud does not actively maintain or provide modules support for all active Spectrocloud Terraform Provider releases / features. 
This repository can be forked and customized to meet your specific requirements.

Reply:
updated